src/pages/UserPosts.jsx

```diff
@@ -10,7 +16,11 @@ const UserPosts = () => {
   const [userPosts, setUserPosts] = useState([]);
   useEffect(() => {
     const postsRef = collection(db, "posts");
-    const q = query(postsRef, where("uid", "==", profileId));
+    const q = query(
+      postsRef,
+      where("uid", "==", profileId),
```

Comment:
Suggestion to move queries to util files for better modularity

Reply:
Done